packages/flutter/lib/src/foundation/_ui_stub.dart

```diff
@@ -0,0 +1,6 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
This file is a little wired because it is not following the convention set by the other _ files.